gravityflow/gravityflow
# Fix Custom File Name to appear under Send by Email on PDF Workflow Step

## Description
The 'Custom file Name' only applies to 'Send by Email' on PDF Workflow Step. However, it appears as a separate Workflow Setting. This PR fixes the custom file name to appear as a sub-setting.

This PR fixes [gravityflow/backlog#143.](https://github.com/gravityflow/backlog/issues/143)

## Testing instructions
1. Create a Workflow PDF Step.
2. Scroll down and note 'Custom File Name' setting can be worked with, even if 'Send by Email' is disabled.
3. Switch to this branch and repeat above steps. Confirm that 'Custom File Name' is now appearing as a sub-setting under 'Send by Email'.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes
N/A

## Checklist:
